ccm_web/server/src/canvas/canvas.interfaces.ts

```diff
@@ -62,8 +67,20 @@ export interface CanvasEnrollment {
   type: UserEnrollmentType
 }
 
+export interface CanvasAccount {
+  id: number
+  name: string
+  parent_account_id: number | null
+}
+
 export type CanvasEntity = CanvasEnrollment | CanvasCourseSection | CanvasCourse
 
+// Composites
+
+export interface CourseWithSections extends CanvasCourseBase {
+  sections: CanvasCourseSection[]
+}
```

Comment:
It was a deliberate choice not to use the `Canvas` prefix here. I thought maybe composite, non-atomic entities might be easier to identify if we didn't use that. Open to changing it though.